aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -225,6 +225,16 @@
    */
   public static final String S3_WRITE_TAGS_PREFIX = "s3.write.tags.";
 
+  /**
+   * Used by {@link S3FileIO} for access point. To set, we can pass a catalog property.
+   * <p>
+   * For more details, see https://aws.amazon.com/s3/features/access-points/
+   * <p>
+   * Example in Spark:
+   *    --conf spark.sql.catalog.test.s3.access-points.my-bucket=arn:aws:s3::123456789012:accesspoint:mfzwi23gnjvgw.mrap
```

Comment:
I would probably not include a Spark-specific example. Instead, just show configuration from the catalog level, like `s3.access-points.my-bucket=...`.